Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +36,63 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            // split the message
+            string[] stringArray = message.Split(Environment.NewLine);
+            int maxLength = -1;
+
+
+            // calculate max length
+            foreach (string item in stringArray)
+            {
+                if (maxLength < item.Length)
+                {
+                    maxLength = item.Length;
+                }
+            }
+
+            // calculate table borders
+            char corner = '+';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'corner'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069037)